drakrun/drakrun/draksetup.py

```diff
@@ -393,57 +409,55 @@ def send_usage_report(report):
         logging.exception("Failed to send usage report. This is not a serious problem.")
 
 
-def create_rekall_profiles(injector: Injector):
+def create_rekall_profile(injector: Injector, file: DLL):
     tmp = None
-
-    for file in dll_file_list:
-        try:
-            logging.info(f"Fetching rekall profile for {file.path}")
-
-            local_dll_path = os.path.join(PROFILE_DIR, file.dest)
-            guest_dll_path = str(PureWindowsPath("C:/", file.path))
-
-            cmd = injector.read_file(guest_dll_path, local_dll_path)
-            out = json.loads(cmd.stdout.decode())
-            if out["Status"] == "Error" and out["Error"] in ["ERROR_FILE_NOT_FOUND", "ERROR_PATH_NOT_FOUND"]:
-                raise FileNotFoundError
-            if out["Status"] != "Success":
-                logging.debug("stderr: " + cmd.stderr.decode())
-                logging.debug(out)
-                # Take care if the error message is changed
-                raise Exception("Some error occurred in injector")
-
-            guid = pdb_guid(local_dll_path)
-            tmp = fetch_pdb(guid["filename"], guid["GUID"], PROFILE_DIR)
-
-            logging.debug("Parsing PDB into JSON profile...")
-            profile = make_pdb_profile(tmp)
-            with open(os.path.join(PROFILE_DIR, f"{file.dest}.json"), 'w') as f:
-                f.write(profile)
-        except json.JSONDecodeError:
-            logging.debug(f"stdout: {cmd.stdout}")
-            logging.debug(f"stderr: {cmd.stderr}")
-            logging.debug(traceback.format_exc())
-            raise Exception(f"Failed to parse json response on {file.path}")
-        except FileNotFoundError:
-            logging.warning(f"Failed to copy file {file.path}, skipping...")
-        except RuntimeError:
-            logging.warning(f"Failed to fetch profile for {file.path}, skipping...")
-        except Exception as e:
+    try:
+        logging.info(f"Fetching rekall profile for {file.path}")
+
+        local_dll_path = os.path.join(PROFILE_DIR, file.dest)
+        guest_dll_path = str(PureWindowsPath("C:/", file.path))
+
+        cmd = injector.read_file(guest_dll_path, local_dll_path)
+        out = json.loads(cmd.stdout.decode())
+        if out["Status"] == "Error" and out["Error"] in ["ERROR_FILE_NOT_FOUND", "ERROR_PATH_NOT_FOUND"]:
+            raise FileNotFoundError
+        if out["Status"] != "Success":
+            logging.debug("stderr: " + cmd.stderr.decode())
+            logging.debug(out)
```

Comment:
warning, otherwise it will be cumbersome when people create issues where logs don't have debug prints